plugins/kubernetes/src/components/PodExecTerminal/PodExecTerminalAttachAddon.ts

```diff
@@ -15,9 +15,9 @@
  */
 import { AttachAddon, IAttachOptions } from 'xterm-addon-attach';
 
-const textEncoder = new TextEncoder();
-
 export class PodExecTerminalAttachAddon extends AttachAddon {
+  #textEncoder = new TextEncoder();
```

Comment:
What's this `#` convention? seen `$` to mean private variables or observables but can't remember seeing this one...

Reply:
This is the _native_ javascript syntax! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields